rust/core-lib/src/view.rs

```diff
@@ -20,17 +20,14 @@ use std::ops::Range;
 use serde_json::Value;
 
 use xi_rope::rope::{Rope, LinesMetric, RopeInfo};
-use xi_rope::delta::{Delta, DeltaRegion};
-use xi_rope::tree::{Cursor, Metric};
+use xi_rope::delta::{Delta};
```

Comment:
No need for braces on a single item import.

You might also try running clippy on this, to make sure we're keeping things clean(ish).